controller/config.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/openziti/fabric/pb/ctrl_pb"
 	"github.com/openziti/fabric/pb/mgmt_pb"
 	"github.com/openziti/fabric/router/xgress"
+	"github.com/openziti/foundation/channel"
 	"github.com/openziti/foundation/channel2"
```

Comment:
I'm not clear on whether the channel2 import should still be here

Reply:
mgmt channel still uses channel2